test/Interop/Cxx/namespace/Inputs/module.modulemap

```diff
@@ -0,0 +1,26 @@
+module Classes {
+  header "classes.h"
+}
+
+module ClassesSecondHeader {
+  header "classes.h"
+  header "classes-second-header.h"
```

Comment:
It does include "classes.h". But both headers are required to be able to use the imported APIs. I'll investigate because we should be able to see the stuff that's "defined in defs" without "importing" both headers.

Reply:
Oh sorry indeed it does. I was thinking of templates-second-header.h.

But yes, it would be best if classes.h (and other first-level headers) could be a part of only one module.

Reply:
So, here's the problem: when we find a declaration, we immediately look for the definition and see if we can import that. This means Swift thinks all of these APIs are part of `Classes` and not `ClassesSecondHeader`. While this is sort of problematic, I think it's at least mostly an unrelated issue. I'd like to keep the tests here with a TODO comment linking to an SR so that the problem can be fixed in its own PR.

Reply:
Sounds good, thank you!